libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -985,6 +1051,7 @@ where
     }
 }
 
+/// Auxiliary method for handling request send to connection manager.
```

Comment:
Doesn't parse - a word or words appear to be missing from "handling request send to". 

Also (as long as you're changing this line) there should be a "the" before "connection manager". This issue occurs in several other comments, too.